modules/end-user-guide/pages/microsoft-visual-studio-code-open-source-ide.adoc

```diff
@@ -0,0 +1,31 @@
+:_content-type: CONCEPT
+:description: Microsoft Visual Studio Code - Open Source IDE
+:keywords: user-guide, selecting, IDE, select, selecting, editor, ide, vs-code, visual-studio, visual-studio-code, jetbrains, intellij, pycharm, theia, che-theia
+:navtitle: Microsoft Visual Studio Code - Open Source
+:page-aliases:
+
+[id="microsoft-visual-studio-code-open-source-ide"]
+= Microsoft Visual Studio Code - Open Source IDE
+
+The {prod-short} build of link:https://github.com/microsoft/vscode[Microsoft Visual Studio Code - Open Source] is the default IDE of a new workspace.
+
+You can have extensions automatically installed in this IDE for workspaces, as explained in the next section, from the xref:administration-guide:extensions-for-microsoft-visual-studio-code-open-source.adoc[Open VSX registry].
+
+TIP: You can use the link:https://code.visualstudio.com/api/ux-guidelines/command-palette[Command Palette] to find and run the commands specified in `devfile.yaml`.
```

Comment:
```suggestion
TIP: Use the link:https://code.visualstudio.com/api/ux-guidelines/command-palette[Command Palette] to find and run the commands specified in `devfile.yaml`.
```

Reply:
Thank you. I'll keep this as `You can` so that it's parallel to the next tip, which requires `You can`.